packages/expo-dev-client-components/src/create-primitive.tsx

```diff
@@ -48,9 +49,20 @@ export function create<T, O extends Options>(
     const selectorStyles = stylesForSelectors(props, config.selectors, { theme });
     const selectorPropsStyles = stylesForSelectorProps(props.selectors, { theme });
 
+    const variantFreeProps = { ...props };
+
+    // @ts-ignore
+    // there could be a conflict between the primitive prop and the variant name
```

Comment:
👁️ @FiberJW i suspect this was causing some issues w/ prop changes to `View`, potentially others in the future